lib/engine/game/g_1848/game.rb

```diff
@@ -740,6 +755,80 @@ def revenue_for(route, stops)
           k_sum = stops.count { |rl| rl.hex.tile.label.to_s == 'K' }
           super + K_BONUS[k_sum]
         end
+
+        # recievership
+
+        def close_corporation(corporation, quiet: false)
+          @close_corp_count += 1
+          @player_corp_close_count[corporation.owner] += 1
+
+          # boe gets all the tokens
+          corporation.tokens.select(&:used).each do |token|
```

Comment:
remove the select and just do each do |token|
next used

Reply:
fixe